website/docs/integrations/overview.md

```diff
@@ -39,8 +39,9 @@ Discover feature flag usages in your source code and upload the found code refer
 Discover feature flag usages in your source code and upload the found code references to ConfigCat.
 [Documentation](integrations/bitrise.md)
 ### Terraform
-Configure and access ConfigCat resources via ConfigCat Public Management API with Terraform.
+Manage feature flags directly from Terraform HCL scripts.
+
 [Documentation](integrations/terraform.md)
 ### Amplitude
-Annotate your Amplitude charts with feature flag changes.
+Annotate your Amplitude charts when a feature flag changes.
```

Comment:
This integration really adds a "feature flag change" annotation to Amplitude charts.
So, I think "with" was the correct word here.

Reply:
agreed